json-logs/samples/api/conversations.list.json

```diff
@@ -125,5 +125,6 @@
   "error": "",
   "needed": "",
   "provided": "",
-  "arg": ""
+  "arg": "",
+  "callstack": ""
```

Comment:
Should the callstack param be left out of these json-logs files? (I don't see it included for any other APIs)